content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,33 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+When scaling down, the ReplicaSet controller chooses which pods to delete by sorting the available pods to
```

Comment:
I would argue that this is too detailed to be useful to users. Also we should avoid locking the criteria in a way that users start relying on it, making it harder to change in the future. In particular, we should state that these are preferences and not something that is guaranteed.

Important downscale selection criteria IMO:
1. Pending (including unscheduled) pods.
2. pod-deletion-cost annotation (probably worth stating the default)
3. too many pods in a node (this is the rank, although not sure if we should include it in the documentation).
4. lower running time (bucketed)

And if they all match, selection is random (the UIDs is 100% an implementation detail that shouldn't go in the documentation).